src/main/java/org/fundacionjala/sfdc/pages/opportunities/OpportunityForm.java

```diff
@@ -268,8 +305,8 @@ public OpportunityForm setPrimaryCampaignSource(final String primaryCampaignSour
      * @return {@link OpportunityForm}
      */
     public OpportunityForm setOrderNumber(final String orderNumber) {
-        orderNumberTextBox.clear();
-        orderNumberTextBox.sendKeys(orderNumber);
+        orderNumberTextField.clear();
+        orderNumberTextField.sendKeys(orderNumber);
```

Comment:
It would be better to use the methods of CommonActions.
- CommonActions.sendKeys(........)